sos/report/__init__.py

```diff
@@ -447,7 +447,7 @@ def display_help(cls, section):
         ssec.add_text(helpln)
 
     def print_header(self):
-        print(f"\n{_(f'sos report (version {__version__})')}\n")
+        self.ui_log.info(f"\n{_(f'sos report (version {__version__})')}\n")
```

Comment:
We might use direct `print` by a purpose here as there is a call flow when `ui_log` might not be set up..?

Do you @TurboTurtle recall anything? If not I am ok with this.